src/chariott_grpc.rs

```diff
@@ -142,7 +142,7 @@ impl<T: Observer + Send + Sync + 'static> ChariottService for ChariottServer<T>
         #[cfg(not(test))]
         let broker = &self.broker;
         #[cfg(test)]
-        _ = self.broker; // Suppress dead code warning when test feature is active.
+        let _ = self.broker; // Suppress dead code warning when test feature is active.
```

Comment:
I based this change off a conversation found here: https://github.com/rust-lang/rust/issues/59144